tm2/pkg/std/zero_address.go

```diff
@@ -0,0 +1,15 @@
+package std
+
+import "github.com/gnolang/gno/tm2/pkg/crypto"
+
+const (
+	zAddressBech32 = string("g100000000000000000000000000000000dnmcnx")
```

Comment:
`g1` is the bech32 HRP (and separator) specific to the gno.land domain.

The `tm2` system should not be aware of gno.land or gno.

Please, remove the changes in `tm2/`.

Reply:
Moved to gno.land
https://github.com/gnolang/gno/pull/2401/commits/57617d88095ae940e9fc8356762e15b2136322ea